doc/deployment/connect-to-cluster.md

```diff
@@ -0,0 +1,187 @@
+If you have already deployed a Pachyderm cluster, you can
+set a Pachyderm IP address by updating your cluster configuration
+file.
+
+This configuration is supported for deployments that do not have
+a firewall set up between the Pachyderm cluster deployed in the cloud
+and the client machine.
+Defining a dedicated `pachd` IP address and host is a more reliable
+way that might also result in better performance compared to
+port-forwarding. Therefore, Pachyderm
+recommends that you use contexts in all production environments.
+
+This configuration requires that you deploy an ingress controller
+and a reliable security method to protect the traffic between the
+Pachyderm pods and your client machine. Remember that exposing your
+traffic through a public ingress might
+create a security issue. Therefore, if you expose your `pachd` endpoint,
+you need to make sure that you take steps to protect the endpoint and
+traffic against common container security threats. Port-forwarding
+might be an alternative which might result in sufficient performance
+if you place the data that is consumed by your pipeline in object
+store buckets located in the same region.
+
+For more information about Pachyderm contexts, see
+[Manage Cluster Access](../managing_pachyderm/connect-to-cluster.html)
+
+To connect by using a Pachyderm context, complete the following
+steps:
+
+1. Get the current context:
+
+   ```bash
+   $ pachctl config get active-context
+   ```
+
+   If no IP address is set up for this cluster, you get the following
+   output:
+
+   ```bash
+   $ pachctl config get context <name>
+   {
+
+   }
+   ```
+
+1. Set up `pachd_address`:
+
+   ```bash
+   $ pachctl config update context <name> --pachd-address <host:port>
+   ```
+
+   **Example:**
+
+   ```bash
+   $ pachctl config update context local --pachd-address 192.168.1.15:30650
+   ```
+
+   **Note:** The `pachd` port is `30650` or `650`.
+
+1. Verify that the configuration has been updated:
+
+   ```bash
+   $ pachctl config get context local
+   {
+      "pachd_address": "192.168.1.15:30650"
+   }
+   ```
+
+## Connect by Using Port-Forwarding
+
+The Pachyderm port-forwarding is the simplest way to enable cluster access
+and test basic Pachyderm functionality. Pachyderm automatically starts
+port-forwarding from `pachctl` to `pachd`. Therefore, the traffic
+from the local machine goes to the `pachd` endpoint through the
+Kubernetes API. However, to open a persistent tunnel to other ports, including
+the Pachyderm dashboard, git and authentication hooks, the built-in HTTP
+file API, and other, you need to run port-forwarding explicitly.
+
+Also, if you are connecting with port-forward, you are using the `0.0.0.0`.
+Therefore, if you are using a proxy, it needs to handle that appropriately.
+
+Although port-forwarding is convenient for testing, for production
+environments, this connection might be too slow and unreliable.
+The speed of the port-forwarded traffic is limited to `1 MB/s`.
```

Comment:
I did not know this!  Where on earth do they set that limit?

Reply:
I found thi here: https://docs.pachyderm.io/en/latest/managing_pachyderm/general_troubleshooting.html?highlight=1MB%2Fs#recourse